operator/config/rbac/kustomization.yaml

```diff
@@ -18,4 +18,13 @@ resources:
 - cnrm_viewer_role.yaml
 - cnrm_viewer_role_binding.yaml
 - role.yaml
-- manager_role_binding.yaml
\ No newline at end of file
+- manager_role_binding.yaml
+
+patches:
```

Comment:
This patch keeps the `cnrm_viewer_role.yaml` labels and annotations and only replace the `.rules` field.

Reply:
Could you provide an example output? I am not sure I follow how it works. In `${REPO_ROOT}/config/installbundle/components/clusterroles/cnrm_viewer.yaml`, we have a role name `viewer`, but in the `cnrm_viewer_role.yaml` the role name is `cnrm-viewer`

Reply:
It looks like the "copy-dependency-manifests" was copying from the rendered manifest which has `cnrm-` prefix already added.

Reply:
Here's the hydrated diff. https://paste.googleplex.com/5782340843077632
"<" is removed, ">" is added.
The ClusterRole name is changed from `cnrm-viewer` to `configconnector-operator-cnrm-viewer`.
Many RBAC are added to new groups.

Reply:
I think my original confusion was because of the `name: viewer` in the patch.